build_defs/multiversion_wheel.build_defs

```diff
@@ -30,7 +30,7 @@ def python_multiversion_wheel(name, version:str, urls:list, licences:list=None,
         name = name,
         srcs = file_rules,
         outs = [name],
-        cmd = 'for SRC in $SRCS; do $TOOL x -o $TMPDIR $SRC; done && rm -rf %s/*.egg-info %s/*.dist-info',
```

Comment:
For some reason the sandbox tool sets $TMPDIR to /tmp. Not sure if that's needed by some core util but it did break this.

Reply:
I think it resets TMPDIR and TMP_DIR so they don't point at the old plz-out directory.

Why does this thing care which it is?

Reply:
Well it's accessing $TMPDIR but that's been set to `/tmp`. I guess it should be `/tmp/please_sandbox` right?